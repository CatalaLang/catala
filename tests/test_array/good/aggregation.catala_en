## Article

```catala
declaration scope A:
   context x content collection money

scope A:
   definition x equals [$0; $4 +$ $5; $8 *$ 0.65]

declaration scope B:
   context a scope A
   context max content money
   context min content money
   context y content money
   context z content integer

scope B:
   definition max equals maximum money initial $0 for m in a.x of m *$ 2.0
   definition min equals minimum money initial $20 for m in a.x of m +$ $5
   definition y equals sum money for m in a.x of (m +$ $1)
   definition z equals number for m in a.x of (m >=$ $8.95)
```
