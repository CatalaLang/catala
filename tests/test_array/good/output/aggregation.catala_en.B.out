[RESULT] Computation successful! Results:
[RESULT] max = $18.00
[RESULT] min = $5.00
[RESULT] y = $17.20
[RESULT] z = 1
